v1.9.0 breaks compatibility with `express.Router`

In v1.8.0, it was possible to pass an express `Router` instance as the first argument to the `listen` function, e.g.:

```typescript
import { Router } from 'express';
import { listen } from 'soap';

const router = Router();

listen(router, ...);
```

In v1.9.0, this now causes a type error:

```
error TS2345: Argument of type 'Router' is not assignable to parameter of type 'ServerType'.
  Type 'IRouter' is missing the following properties from type 'IExpressApp': setTimeout, maxHeadersCount, maxRequestsPerSocket, timeout, and 30 more.
```

This is due to the changes in https://github.com/vpulim/node-soap/pull/1387 where the `IExpressApp` type was changed to extend `http.Server`.

In fact, copilot has commented on the PR that this is incorrect: https://github.com/vpulim/node-soap/pull/1387/changes#r2834770251

--- a/src/server.ts
+++ b/src/server.ts
@@ -10,7 +10,8 @@ import { WSDL } from './wsdl';
 import { BindingElement, IPort } from './wsdl/elements';
 import zlib from 'zlib';
 
-interface IExpressApp extends http.Server {
+interface IExpressApp {
+  address?: () => { address: string; port: number; family: string } | string;
   route;
   use;
 }

